Sources/FrontEnd/BundledNode+Extensions.swift

```diff
@@ -44,3 +44,21 @@ extension BundledNode where T == SequenceExpr.ID, P == TypedProgram {
   }
 
 }
+
+extension BundledNode where T == FunctionDecl.ID, P == TypedProgram {
```

Comment:
```suggestion
extension BundledNode where T == FunctionDecl.ID {
```
I think the property lives in the AST, so you should be able to compute it with any kind of program. Same for `externalName`

Reply:
Ah, now I see that you defined property maps for those attributes.